doc/source/new.rst

```diff
@@ -4,6 +4,15 @@ What's New
 Releases
 -------------
 
+v1.4.0 (May 2025)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Release 1.4.0
+- Transition to CMake build using scikit-build-core
+- Packaging updates and adoption of pyproject.toml
+- Support for Python 3.12
```

Comment:
Is there anything preventing 3.13 from working as well?  Maybe seek that in too if there's no code changes necessary?

Reply:
I don't wish to hold up the release, thanks for trying. c/f conda-forge/basemap-feedstock#143

Reply:
Thanks for logging / looking into that!  

I'll drop the 3.13 for now, but happy to revisit later.